front_end/html/profile_courses.html

```diff
@@ -53,9 +53,9 @@ <h2 style="display: inline">My Courses</h2>
                                 <div class="media-content">
```

Comment:
These changes address long course titles spilling over on the course registration page.